packages/core-data/src/entity-types/term.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import type { Context, ContextualField, OmitNevers } from './helpers';
+
+import type { BaseEntityRecords as _BaseEntityRecords } from './base-entity-records';
+
+declare module './base-entity-records' {
+	export namespace BaseEntityRecords {
+		export interface Term< C extends Context > {
+			/**
+			 * A human-readable description of the term.
+			 */
+			description: ContextualField< string, 'view' | 'edit', C >;
+			/**
+			 * The title for the term.
+			 */
+			name: string;
```

Comment:
According to the [REST API docs](https://developer.wordpress.org/rest-api/reference/categories/#schema-name), this can be available in multiple contexts:

```suggestion
			name: ContextualField< string, 'view' | 'embed' | 'edit', C >;
```